retrofit-mock/src/main/java/retrofit/mock/Behavior.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (C) 2013 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package retrofit.mock;
+
+import java.io.IOException;
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+
+public final class Behavior {
+  private static final int DEFAULT_DELAY_MS = 2000; // Network calls will take 2 seconds.
+  private static final int DEFAULT_VARIANCE_PERCENTT = 40; // Network delay varies by ±40%.
+  private static final int DEFAULT_FAILURE_PERCENTT = 3; // 3% of network calls will fail.
+
+  public static Behavior create() {
+    return new Behavior(new Random());
```

Comment:
I'd prefer to keep it truly random as we want it to exercise as much unpredictability as the real world by default. You cannot rely on mobile networks to behave just like you cannot rely on getting the same behavior over and over.

Unless you want to, of course, which is why there's the overload which lets you seed (like for tests).